pep257.py

```diff
@@ -377,8 +377,15 @@ def parse_all(self):
             elif (self.current.kind == tk.STRING or
                   self.current.value == ','):
                 all_content += self.current.value
+            elif self.current.kind == tk.NAME:
+                all_content += "'%s'" % self.current.value
```

Comment:
It was in https://github.com/GreenSteam/pep257/pull/126 , where I broke out this change in particular.